src/legacy/wildwest.rs

```diff
@@ -67,6 +67,33 @@ impl From<FrontmatterBuilder> for frontmatter::FrontmatterBuilder {
     }
 }
 
+#[derive(Debug, Eq, PartialEq, Hash, Copy, Clone)]
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub enum SassOutputStyle {
+    Nested,
+    Expanded,
+    Compact,
+    Compressed,
+}
+
+#[derive(Debug, PartialEq)]
+#[derive(Serialize, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct SassOptions {
```

Comment:
Someone might ask why duplicate `SassOutputStyle` and `SassOptions` rather than load it from `config`.

The benefit is that we can freely evolve the `config` version after this is submitted and released while we won't be able to change this version after release unless we are breaking compatibility and don't feel the upgrade burden will be too much.